common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -251,15 +251,21 @@ describe 'Problem', ->
               callable()
             done: (callable) ->
               callable()
-        spyOn @problem, 'enableAllButtons'
         @problem.submit()
-        expect(@problem.enableAllButtons).toHaveBeenCalledWith false, true
         if jQuery.active == 0
           deferred.resolve()
         deferred.promise()
 
+      spyOn(@problem, 'disableButtons').and.callThrough()
```

Comment:
For this test specifically, it's testing that the buttons are disabled immediately following the submit() call. Submit has both the disabling + enabling of the buttons, so during the execution of submit the button will go from Enabled + Submit, Disabled + Submitting, Enabled + Submit. Is there a way in Jasmine to test that middle state, rather than just start and end? Or is that something that might be better suited as an acceptance test?